app/routes/screens/main/race/mapDispatchToProps.js

```diff
@@ -0,0 +1,8 @@
+import { bindActionCreators } from "redux";
+import * as geoloactionActions from "../../../../store/actions/geolocation";
+
+const mapDispatchToProps = dispatch => ({
+  boundActionCreators: bindActionCreators(geoloactionActions, dispatch)
```

Comment:
https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options

[mapDispatchToProps(dispatch, [ownProps]): dispatchProps] (Object or Function): **If an object is passed, each function inside it is assumed to be a Redux action creator**. An object with the same function names, but with every action creator wrapped into a dispatch call so they may be invoked directly, will be merged into the component’s props.

If a function is passed, it will be given dispatch as the first parameter. It’s up to you to return an object that somehow uses dispatch to bind action creators in your own way. (Tip: you may use the bindActionCreators() helper from Redux.)

If your mapDispatchToProps function is declared as taking two parameters, it will be called with dispatch as the first parameter and the props passed to the connected component as the second parameter, and will be re-invoked whenever the connected component receives new props. (The second parameter is normally referred to as ownProps by convention.)